install_config/cluster_metrics.adoc

```diff
@@ -188,6 +247,8 @@ xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[p
 volume claims] is handled by the
 xref:../install_config/cluster_metrics.adoc#metrics-deployer[Metrics Deployer].
 
+{product-title} metrics supports also https://github.com/openshift/origin-metrics/blob/master/metrics.yaml#L130[dynamically provisioned persistent volumes].  To use this feature with {product-title} metrics, it is necessary to add an additional flag to the metrics-deployer: `DYNAMICALLY_PROVISION_STORAGE=true`.  At this time, EBS, GCE and Cinder storage backends can be used in order to dynamically provision volumes.  For more information about Dynamically Provisioned PVs, refer to https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/dynamically_provisioning_pvs.html[Dynamically Provisioned PVS].
```

Comment:
that whole bit I mentioned on the other PR about non-versioned self references applies.